aiohttp.svg

```diff
@@ -0,0 +1,2144 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
This is not used/needed anywhere. Seems totally unrelated.

Reply:
I'm totally sorry for this extra file, this is a mistake of mine :/

Reply:
deleted it.